opdracht-one.js

```diff
@@ -0,0 +1,70 @@
+
+// Opdracht 1a: Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de console.
+// const tvName = (array) => {
+//      let tvNamesBe = [];
+//     for (let i = 0 ; i < array.length ; i++) {
+//            tvNamesBe.push(array[i].name);
+//     }
+//     return tvNamesBe;
+// }
+// console.log(tvName(inventory));
+
+//map method itererts over length of array
+//for every entry an anonymous function is carried on
+//makes an new array on the basis of return statement
+//parameter contains the full entry of that loop
+//original array is not changed.
+
+const tvNames = inventory.map((names)=>{
+    return names.name
+})
+console.log(tvNames);
+
+//Opdracht 1b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn. Log de uitkomst in de console.
+// const soldOutTv = (arr) => {
+//     const newSoldOut = [];
+//     for(let i = 0 ; i < arr.length ; i++){
+//         if(arr[i].originalStock === arr[i].sold) {
+//             newSoldOut.push(arr[i]);
+//         }
+//         }
+//         return newSoldOut;
+// }
+//     console.log (soldOutTv(inventory));
+
+const soldOutTv = inventory.filter((soldOut)=>{
+    if (soldOut.originalStock === soldOut.sold){
+        return true;
+    }
+    return false
+})
```

Comment:
Dit zou je wat simpeler kunnen doen
```suggestion
const soldOutTv = inventory.filter(soldOut => soldOut.originalStock === soldOut.sold)
```